modules/installation/main.tf

```diff
@@ -1,8 +1,8 @@
 terraform {
   required_providers {
     yba = {
-      version = "~> 0.1.0"
-      source  = "terraform.yugabyte.com/platform/yba"
+      version = "~> 0.1.6"
+      source  = "yugabyte/yba"
```

Comment:
Are we restricting the installation resource to the one on the registry since it's already a deprecated resource and would not have any further changes?

Reply:
Sorry, I did not understand your question. Without this change, `terraform init` failed with this error.
```
� Error: Failed to query available provider packages
�
� Could not retrieve the list of available versions for provider terraform.yugabyte.com/platform/yba: could not connect to terraform.yugabyte.com: failed to request discovery document: Get "https://terraform.yugabyte.com/.well-known/terraform.json": dial tcp: lookup terraform.yugabyte.com on 127.0.0.1:53: no such
� host
�
```

Reply:
And we may have to make the same fix for all other `main.tf` files in the modules. I made this change directly on GitHub for now, as I wanted to use this one and was blocked.